src/modules/card/card.service.ts

```diff
@@ -514,4 +515,134 @@ export class CardService {
     evidences.forEach((level) => evidencesMap.set(level.id, level));
     return evidencesMap;
   };
+
+  findSiteCardsGroupedByPreclassifier = async (siteId: number) => {
+    try {
+      const rawPreclassifiers = await this.cardRepository
+        .createQueryBuilder('card')
+        .select([
+          "CONCAT(card.preclassifier_code, ' ', card.preclassifier_description) AS preclassifier",
```

Comment:
can we move this into a QueryConstants file?

Reply:
okay